Documentation/running-clair.md

```diff
@@ -41,17 +41,19 @@ A [PostgreSQL 9.4+] database instance is required for all instructions.
 
 ### Cluster
 
-#### Kubernetes
+#### Kubernetes (Helm)
 
 If you don't have a local Kubernetes cluster already, check out [minikube].
+This assumes you've already ran helm init and you have access to a currently running instance of Tiller.
```

Comment:
Put `helm init` in upticks so that it stands out as a command.
```
`helm init`
```

Reply:
yikes, good catch